src/init/prepare.sh

```diff
@@ -52,6 +52,9 @@ getEnv() {
     # allow limiting log pollution on startup
     silent_init=${SILENT_INIT:-false}
     dev_mode=${DEV_MODE:-false}
+    auto_migration=${AUTO_MIGRATION:-false}
+    skip_db_init=${SKIP_DB_INIT:-true}
+    migration_data_dir=${MIGRATION_DATA_DIR:-/elabftw/migration_state}
```

Comment:
Actually I would call it `auto-db-init.lock` then. Because that is what it really is. The db:upgrade command is always run if `auto_db_update` is set to true. I use that file only to remember that the initial database install has already been executed.
Storing the file in `/run` could work but I am not sure about possible side effects. Do other parts of elabFTW use the `/run` folder? Because in my installation I mount this directory as an external volume so that the lock file is preserved between container runs. Using the `/run` folder could either shadow contents of the base image or also preserve other unintended files. But if you prefer that I can definitly set `/run` as the default value.

Reply:
I'm thinking a better approach would be to have the `install start` command be idempotent and exit 0 if it's already installed/initialized. This would remove the need of a state file. Also, it's pretty easy to know if the db is already installed. In fact I thought that the install command already did that but it doesn't.

Reply:
Now it is idempotent: https://github.com/elabftw/elabftw/commit/07ad412ec2b6bcd639105a6453d6e1f510a8707e